### ISSUE TITLE
List Properies UI focus glitch

### ISSUE THREAD

**Issue by niegowski** [ckeditor/ckeditor5#12825]
## 📝 Provide detailed reproduction steps (if any)

List properties UI: 

![image](https://user-images.githubusercontent.com/1232187/200878440-ed934297-1cba-47c1-896b-e2b8a1ebb790.png)

**Comment by Dumluregn** [comment]
Interestingly for me `Start at` is at the top:

![](https://user-images.githubusercontent.com/10883184/201064958-b38a93b4-2cd8-4e9d-bd62-0f1ebfccc264.png)

Chrome 107, both docs and manual test.

**Comment by niegowski** [comment]
> Interestingly for me `Start at` is at the top:

Maybe because I opened the list style dropdown from the "show more items" button in the all-features manual test.

**Comment by Dumluregn** [comment]
Right, same here. So the styles are different for the stuff nested in the "show more items" dropdown.

**Comment by scofalik** [comment]
+1, very annoying, I just came here to post this issue.
